drivers/sensor/icm42670S/icm42670S.c

```diff
@@ -224,6 +231,31 @@ static int icm42670S_channel_get(const struct device *dev,
 		val->val2 = data->wom_y;
 		val ++;
 		val->val1 = data->wom_z;
+#endif
+#ifdef CONFIG_ICM42670S_AML_POINTING
+	} else if ((enum sensor_channel_icm42670S)chan == SENSOR_CHAN_AML) {
+		val->val1 = data->delta[0];
+		val->val2 = data->delta[1];
```

Comment:
This is not correct according to https://docs.zephyrproject.org/latest/hardware/peripherals/sensor.html#c.sensor_value
val2 is supposed to be the decimal part.

Reply:
Good catch I didn't know I wasn't allowed to use the structure differently. I will need to do the change also for APEX features outputs